### DISCUSSION TITLE
Discord canary now supports wayland screensharing even with audio

### DISCUSSION THREAD

**Discussion by Twig6943** [gianklug/wearewaylandnow#81] [General]
title.

**Comment by gianklug** [comment]
let's wait for this to be on the stable release channel first, afterwards I'll update the page.

**Comment by KosmX** [comment]
Actually, I have bad news :(  
Yes, discord can use Wayland, and discord can screen-capture on wayland. But discord (probably for keylogging) still uses X11 and cannot run on pure wayland environment.  

Although, you can run discord with xvfb (dummy X11 server), that way discord will be able to keylog the nothing while drawing a functional window on Wayland

Please note that when I tested, discord use keylogging only for shortcuts, and I saw no keylogging telemetry, but it is a closed-source software.

**Comment by gianklug** [comment]
I've recently tested this out again on stable; sadly still doesn't work directly.

I'd recommend everyone that doesn't rely on PTT support to switch to Vesktop.
